### DISCUSSION TITLE
v1.11.0

### DISCUSSION THREAD

**Discussion by GeckoEidechse** [R2Northstar/Northstar#391] [Release]
![regions screenshot](https://user-images.githubusercontent.com/64006268/205449342-5f1a5c13-fa7a-4c5d-beea-85b77934ed22.png)

![log colour screenshot](https://user-images.githubusercontent.com/40122905/207134398-c9d843c3-8cb6-4cf6-bf04-0a7b3e5884bd.png)

**Features:**
Display server region in serverbrowser ([#479](https://github.com/R2Northstar/NorthstarMods/pull/479))
Parse and expose region field from JSON server list ([#232](https://github.com/R2Northstar/NorthstarLauncher/pull/232))
Logging improvements + Colours (new) ([#320](https://github.com/R2Northstar/NorthstarLauncher/pull/320))
Squirrel bridge v3 ([#310](https://github.com/R2Northstar/NorthstarLauncher/pull/310))
Allow `kb_act.lst` to be compiled from multiple mods ([#298](https://github.com/R2Northstar/NorthstarLauncher/pull/298))
Added platform information to User-Agent ([#367](https://github.com/R2Northstar/NorthstarLauncher/pull/367))
Localize stringcommands to allow servers to use localised messages ([#535](https://github.com/R2Northstar/NorthstarMods/pull/535))
Added support for hiding the `[SERVER]` tag in server messages ([#506](https://github.com/R2Northstar/NorthstarMods/pull/506))
Added customiseable eject quotes ([#527](https://github.com/R2Northstar/NorthstarMods/pull/527))
Added entity functions for reading userinfo cvars to CPlayer class ([#521](https://github.com/R2Northstar/NorthstarMods/pull/521))
Added support for querying player userinfo cvars in script ([#293](https://github.com/R2Northstar/NorthstarLauncher/pull/293))
Squirrel functions auto bind ([#299](https://github.com/R2Northstar/NorthstarLauncher/pull/299), [#327](https://github.com/R2Northstar/NorthstarLauncher/pull/327))
Allow vmts do be loaded from disk ([#292](https://github.com/R2Northstar/NorthstarLauncher/pull/292))
Load `sp_` rpaks on dedi (coop) ([#283](https://github.com/R2Northstar/NorthstarLauncher/pull/283))
Added KeyValues class ([#303](https://github.com/R2Northstar/NorthstarLauncher/pull/303))

**Changes:**
Improve mouse capture handling in menus ([#520](https://github.com/R2Northstar/NorthstarMods/pull/520))
Removed launcher proxy ([#363](https://github.com/R2Northstar/NorthstarLauncher/pull/363))
Move include directory ([#337](https://github.com/R2Northstar/NorthstarLauncher/pull/337))
Use new system for adding keybinds to `kb_act.lst` ([#529](https://github.com/R2Northstar/NorthstarMods/pull/529))
New syntax for reversed structs ([#324](https://github.com/R2Northstar/NorthstarLauncher/pull/324))
Make most mod-setting related funcs global ([#364](https://github.com/R2Northstar/NorthstarLauncher/pull/364))

**Fixes:**
New crashhandler ([#325](https://github.com/R2Northstar/NorthstarLauncher/pull/325))
Don't set initial dedicated server title to the profile name ([#352](https://github.com/R2Northstar/NorthstarLauncher/pull/352))
Update exit value in hooks to valid code ([#354](https://github.com/R2Northstar/NorthstarLauncher/pull/354))
Exit instead of abort on server compile error ([#353](https://github.com/R2Northstar/NorthstarLauncher/pull/353))
Fixed script compile errors not being flushed to log file ([#342](https://github.com/R2Northstar/NorthstarLauncher/pull/342))
Fixed broken chat hooks due to missing `AUTOHOOK_DISPATCH()` ([#335](https://github.com/R2Northstar/NorthstarLauncher/pull/335))
Fixed logging ([#333](https://github.com/R2Northstar/NorthstarLauncher/pull/333))
Fixed bug with crashhandler ([#297](https://github.com/R2Northstar/NorthstarLauncher/pull/297))
Removed dev-only flag from `save_enable` ([#315](https://github.com/R2Northstar/NorthstarLauncher/pull/315))
Fixed UI VM Destruction ([#349](https://github.com/R2Northstar/NorthstarLauncher/pull/349))
Removed `sv_cheats` requirement from `r_drawviewmodel` ([#328](https://github.com/R2Northstar/NorthstarLauncher/pull/328))
Removed dev only from `mat_colorcorrection` ([#326](https://github.com/R2Northstar/NorthstarLauncher/pull/326))
Pre-init squirrel managers ([#348](https://github.com/R2Northstar/NorthstarLauncher/pull/348))
Fixed Boost Animations ([#511](https://github.com/R2Northstar/NorthstarMods/pull/511))
Removed unnecessary root table setting and make functions global ([#534](https://github.com/R2Northstar/NorthstarMods/pull/534))
Added missing unreachable to stop compile error ([#528](https://github.com/R2Northstar/NorthstarMods/pull/528))
Fixed `DecodeJSON()` not returning `SQRESULT_NOTNULL` ([#330](https://github.com/R2Northstar/NorthstarLauncher/pull/330))
Fixed slow keyvalues building ([#331](https://github.com/R2Northstar/NorthstarLauncher/pull/331))
Restore `-waitfordebugger` flag ([#321](https://github.com/R2Northstar/NorthstarLauncher/pull/321))
Log function lookup failure on Call ([#355](https://github.com/R2Northstar/NorthstarLauncher/pull/355))

**Contributors:**
@ASpoonPlaysGames @BobTheBob9 @DBmaoha @emma-miler @Erlite @GeckoEidechse @H0L0theBard @JMM889901 @pg9182 @RoyalBlue1 @wolf109909

<hr /><em>This discussion was created from the release <a href='https://github.com/R2Northstar/Northstar/releases/tag/v1.11.0'>v1.11.0</a>.</em>

**Comment by F1F7Y** [comment]
This i pretty poggers
